src/gui/tray/UserLine.qml

```diff
@@ -109,6 +113,8 @@ MenuItem {
                         }
                         Label {
                             id: userStatusMessage
+                            visible: model.isConnected 
+                                     && model.serverUserStatus

```

Comment:
nitpick: for consistency with your other changes above the `&&` should be at the end of the previous line.

Another option is to put them all on a single line, there are only two clauses after all.

I don't mind much where it is in the end, it's really about internal consistency here. :-)